client/components/Signup.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react';
```

Comment:
This file is also using a mix of sans serif and serif fonts.

Reply:
There is still a mixture of Flora's font and a default one. Will address that at some point for sure.